download/download.html.haml

```diff
@@ -217,3 +231,6 @@ title: Download
   %div(id="execution-server-older" class="tab-pane fade")
     :asciidoc
       For older community releases, check https://download.jboss.org/drools/release/[the release archive].
+  %div(id="optaweb-employee-rostering-older" class="tab-pane fade")
+    :asciidoc
+      For older community releases, check https://download.jboss.org/drools/release/[the release archive].
```

Comment:
The drools-planner releases stem from 2012. It's time to stop mentioning those. Let's get rid of all mentions of "For older community releases" (which haven't any for the exe server of optaweb or workbench btw).